app/src/main/java/sdp/moneyrun/ui/menu/MenuActivity.java

```diff
@@ -273,5 +273,9 @@ public void setWeatherFieldsToday(@NonNull WeatherReport report) {
         weatherTypeText.setText(report.getWeatherType());
     }
 
-
+    @Override
+    public void onBackPressed() {
+        // disable the back button from menu since the user should not be able to log in again once logged in properly
```

Comment:
Can you either implement or delete this function? It doesn't seem to do anything, no?

Reply:
IT is not supposed to do anything it disables the back button